astroquery/gaia/core.py

```diff
@@ -911,5 +916,25 @@ def launch_job_async(self, query, name=None, output_file=None,
                                         upload_table_name=upload_table_name,
                                         autorun=autorun)
 
+    def get_status_messages(self):
+        """Retrieve the messages to inform users about
+        the status of Gaia TAP
+        """
+        try:
+            print("parsing notification messages")
```

Comment:
I know that many users don't like default verbosity. OTOH from the archive's side, I understand why these messages are desirable. 
So maybe a middle ground here to add another sentence to say how to turn it off?

Also, add a short paragraph about it in the narrative docs, users should know how to turn this off.

Reply:
This call to our TAP will tell the user if the TAP is under maintenance or it is not working, so they know if what they will do will be working in advance. But it is ok, we can add in the documentation how to turn it off. @mhsarmiento, could you please add this in the documentation of the python file and the .rst file?